client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx

```diff
@@ -334,18 +338,25 @@ class TransferOtherUser extends React.Component {
 	}
 
 	filterAvailableUsers( users ) {
-		return users.filter(
-			( user ) =>
-				includes( user.roles, 'administrator' ) &&
-				this.getWpcomUserId( user ) !== this.props.currentUser.ID
-		);
+		return users.filter( ( user ) => this.getWpcomUserId( user ) !== this.props.currentUser.ID );
```

Comment:
I think there is a pre-existing bug here that's worth fixing. We should exclude also site users who don't have a wpcom account (`linked_user_ID === false`) because we can't transfer a domain to someone who's not a wpcom user.

The UI that lets you transfer the site ownership to someone else, very similar to domain transfer in principle, has code that excludes these users correctly:

https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/site-settings/manage-connection/site-ownership.jsx#L46-L53

See also my another observation about checking `linked_user_ID`: https://github.com/Automattic/wp-calypso/commit/2c332d34027d97ec1da28bbe564b4c618fd2ec95#r49403087

Reply:
Good catch. Fixed in https://github.com/Automattic/wp-calypso/pull/51809/commits/e360ace6cc119d684a1b9af75029587c9416923e